src/components/Tabs/Tabs.tsx

```diff
@@ -22,14 +23,14 @@ const Tabs: FC<TabsProp> = ({ tabs }) => {
       <ReactTabs>
         <TabList>
           {tabs.map(tab => (
-            <ReactTab>
+            <ReactTab key={shortid.generate()}>
```

Comment:
While this technically works, it's inefficient because it will generate a different id on every re-render, which means React will destroy this block, even when the contents are exactly the same. I suggested using the index only because we're using a hard-coded array to map over. However, I'm not that concerned about our performance here, so this works. 👍